app/views/campaigns/overview.html.haml

```diff
@@ -1,6 +1,7 @@
 - content_for :before_title, "#{@campaign.title} Overview  — "
```

Comment:
Let's not make any changes to overview.html.haml for this PR; it'll just affect the ores_plot and alerts tabs.

Reply:
I took out a few lines of the code, line number 41 to 49 from overview.html.haml file that relates to the button Disable/Enable account request.  I will add those lines.